helper/metricsutil/gauge_process.go

```diff
@@ -197,16 +197,18 @@ func (p *GaugeCollectionProcess) collectAndFilterGauges() {
 	p.streamGaugesToSink(values)
 }
 
+// batchSize is the number of metrics to be sent per tick duration.
+const batchSize = 25
```

Comment:
lovely

Reply:
I think this should depend on the default value here:
https://github.com/hashicorp/vault/blob/5f590d4d505dcb7f0021893c23ce2ab49f204d11/internalshared/configutil/telemetry.go#L29

Reply:
I just lifted the local var (`batchSize := 25`) from `streamGuagesToSink` and made it a package-level const.